CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing
+
+This repository includes a [GitHub Action][3] that builds the
+colorschemes once a week. This keeps the colorschemes up-to-date
+automatically.
+
+## Building
+
+### Dependencies
+
+- `>=0.2.0` [base16-builder-go][1]
+- golang `>=1.16` to build base16-builder-go
+
+### Usage for template editing
+
+1. Clone [base16-builder-go][1] somewhere on your system.
+1. Run `cd /path/to/base-builder-go && go build` to generate a binary:
+`/path/to/base-builder-go/base16-builder-go`
+1. Now execute the binary you generated while giving the `-template-dir`
+arg the path to `base16-xfce4-terminal` repository: `./base16-builder-go
+-template-dir ../base16-xfce4-terminal`
+
+Or the above steps represented in shell commands:
+
+```shell 
+cd /path/to/base16-xfce4-terminal/../ # This repos parent dir 
+git clone git@github.com:base16-project/base16-builder-go.git
+cd base16-builder-go
+go build ./base16-builder-go/base16-builder-go \
+  -template-dir ../base16-xfce4-terminal
```

Comment:
I think `-template-dir` is an argument to `base16-builder-go` instead of the `go` command.
But probably we should just suggest using `go install github.com/base16-project/base16-builder-go@latest` to grab the builder instead once https://github.com/base16-project/base16-builder-go/issues/24 is fixed

Reply:
Good catch - this was probably a `go run` call before. I've fixed the linked issue.

Reply:
Is there a simple way of running this with `go install`?

```
go install github.com/base16-project/base16-builder-go@latest
cd $HOME/go/pkg/mod/github.com/base16-project/base16-builder-go@v0.2.7
git clone https://github.com/base16-project/base16-schemes.git schemes # This gives me a permission error
go build
cd -
$HOME/go/pkg/mod/github.com/base16-project/base16-builder-go@v0.2.7/base16-builder-go -template-dir ../base16-xfce4-terminal
```

This isn't working for me since I have the permission error cloning base16-schemes, but is there a better way to do this locally?

Reply:
you don't need to `cd $HOME/go/pkg/mod/github.com/base16-project/base16-builder-go@v0.2.7`. This directory is supposed to be managed by the go tool.
After go install you should have the base16-builder-go command in `~/go/bin/base16-builder-go`

Reply:
Go install seems to be broken for now because I include the schemes using a submodule and go install doesn't clone submodules. I've filed a ticket to look at other options.

Reply:
Maybe we could suggest downloading one of the pre-compiled releases? I do publish them as binaries in the base16-builder-go repo.

Reply:
I don't have specific concerns about pre-built binaries.

But considering the steps above are clear enough, maybe we could drop the shell commands altogether...

Reply:
I've updated `CONTRIBUTION.md`. I've left shell commands to make it more explicit for less technically inclined people, but it's quite brief now.